config/locales/en/course/forum/topics.yml

```diff
@@ -17,3 +17,27 @@ en:
         destroy:
           success: 'Topic %{title} was deleted.'
           failure: 'Failed to delete topic: %{error}'
+        subscribe:
+          tag: 'Subscribe'
+          success: 'You have %{state} Topic %{title}.'
```

Comment:
Actually do we need to tell the users the topic's name when you success to subscribe a topic?